docs/rootfs/etc/caddy/Caddyfile

```diff
@@ -0,0 +1,9 @@
+:2015 {
+  root /public
+  gzip {
```

Comment:
we ran into issues w/ this `gzip` block affecting some image assets... removed in the base Caddyfile in https://github.com/helm/helm-www/pull/43

we may want to do same here (or keep this in mind in future)